test/tests/remote.js

```diff
@@ -163,6 +164,63 @@ describe("Remote", function() {
     });
   });
 
+  it("can fetch from a private repository", function() {
+    this.timeout(15000);
+
+    var repo = this.repository;
+    var remote = Remote.create(repo, "private", privateUrl);
+    var fetchOptions = {
+      callbacks: {
+        credentials: function(url, userName) {
+          return NodeGit.Cred.sshKeyNew(
+            userName,
+            path.resolve("./test/nodegit-test-rsa.pub"),
+            path.resolve("./test/nodegit-test-rsa"),
+            ""
+          );
+        },
+        certificateCheck: function() {
+          return 1;
+        }
+      }
+    };
+
+    return remote.fetch(null, fetchOptions, "Fetch from private")
+      .catch(function() {
+        assert.fail("Unable to fetch from private repository");
+      });
+  });
+
+  it("can reject fetching from private repository without valid " +
```

Comment:
Changed so it that function () { is on the next line, it's necessary to split up the it("string" and the function definition. What's preferable? it("", \nfunction\n{ or it("", \nfunction() { ?